include/odp/api/spec/schedule.h

```diff
@@ -45,6 +45,110 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration
+ * When schedule configuration is not called by the application, the scheduler
+ * behaves in flow-unaware mode and event flow hash values are ignored
+ */
```

Comment:
Flow stuff needs to be removed from config_t intro. The same config is/will be used for many things - flow is only one of those.

Reply:
done